service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -53,6 +56,19 @@ public void start() {
 
         Spark.port(port());
 
+        redirect.any("/", "/health", Redirect.Status.MOVED_PERMANENTLY);
```

Comment:
Ok, straightforward question:
should / return something? IMHO - should, at least "you are doing something wrong, pls, use blah-blah". 

404 - that's really confusing and looks like something wrong...

But not a big deal - we can leave the root page empty.